examples/data-sources/b1ddi_dns_auth_zones/data-source.tf

```diff
@@ -16,3 +16,18 @@ data "b1ddi_dns_auth_zones" "example_auth_zone" {
 
 # Get all DNS Auth Zones
 data "b1ddi_dns_auth_zones" "all_dns_auth_zones" {}
+
+# Get DNS Auth Zone with the specific tags
+data "b1ddi_dns_auth_zones" "all_auth_zones_with_tags" {
+       tfilters = {
+            TestType  = "Acceptance"
+          }
+}
+
+output "zones" {
+  value= data.b1ddi_dns_auth_zones.all_auth_zones.results
+}
+
+output "zones_with_tags" {
+  value= data.b1ddi_dns_auth_zones.all_auth_zones_with_tags.results
+}
```

Comment:
Why do we have this block here but not in the previous object?

Reply:
Fixed